libs/platform/src/lib/components/list/standard-list-item/standard-list-item.module.ts

```diff
@@ -0,0 +1,35 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { FormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import {
+    IconModule, FormModule, CheckboxModule, RadioModule,
+    ToolbarModule, ButtonModule,
+    ListModule, AvatarModule
+} from '@fundamental-ngx/core';
+import { PlatformLinkModule } from '../../link/link.module';
+import { StandardListItemComponent } from './standard-list-item.component';
+import { ListItemDef } from '../base-list-item';
+import { PlatformObjectStatusModule } from '../../object-status/object-status.module';
+
+@NgModule({
+    declarations: [StandardListItemComponent, ListItemDef],
+    imports: [
+        CommonModule,
+        IconModule,
+        FormsModule,
+        CheckboxModule,
+        FormModule,
+        RadioModule,
+        ButtonModule,
+        PlatformLinkModule,
+        ToolbarModule,
```

Comment:
Why list items needs all these modules, when they are imported on list level ?

Reply:
StandardListModule has imports used for it for example
 radio module for single select
 Button Module for LoadMore, delete, edit these are specific.

 Toolbar I removed and placed in ListModule  is used for all the variants.
Did the same for other imports where ever applicable.